packages/core/src/components/Form.tsx

```diff
@@ -725,7 +725,11 @@ export default class Form<
         errors = merged.errors;
       }
       if (focusOnFirstError) {
-        this.focusOnError(schemaValidation.errors[0]);
+        if (typeof focusOnFirstError === 'function') {
+          focusOnFirstError(schemaValidation.errors[0]);
```

Comment:
Not sure but it could be better to just provide the entire error array here. Though this would change the definition of `onFirstError`.

Reply:
yes, that would change the definition.